app/src/main/java/com/duckduckgo/app/browser/TabSwitcherButton.kt

```diff
@@ -20,32 +20,26 @@ import android.animation.Animator
 import android.animation.AnimatorListenerAdapter
 import android.content.Context
 import android.util.AttributeSet
-import android.view.View
+import android.view.LayoutInflater
 import android.widget.RelativeLayout
-import kotlinx.android.synthetic.main.view_tab_switcher_button.view.*
+import com.duckduckgo.app.browser.databinding.ViewTabSwitcherButtonBinding
 
 class TabSwitcherButton @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
 ) : RelativeLayout(context, attrs, defStyleAttr) {
 
+    private val binding = ViewTabSwitcherButtonBinding.inflate(LayoutInflater.from(context), this)
+
     var count = 0
         set(value) {
             field = value
             val text = if (count < 100) "$count" else "~"
-            tabCount.text = text
+            binding.tabCount.text = text
         }
 
     var hasUnread = false
-        set(value) {
-            field = value
-        }
-
-    override fun onFinishInflate() {
-        super.onFinishInflate()
-        View.inflate(context, R.layout.view_tab_switcher_button, this)
-    }
```

Comment:
Could this cause issues now that the View is inflated before onFinishInflate()? I tested it and seems to work fine but there may be an edge case that I am missing?

Reply:
Thanks, I'm trying to understand why we need to use `onFinishInflate` but I don't see any reason. 🤷‍♀️

Reply:
Since the purpose of this task was just to migrate to ViewBinding, I've rolled back changes introduced here. I still think that's not really necessary, but we can address that in the future.